### ISSUE TITLE
Galaxy Classification - Optimizing sequential model accuracy

### ISSUE THREAD

**Issue by wooo0214** [wooo0214/MyTools#3]
### Overview
With kernel_size = (9,9); strides = (4,4), the accuracy gets improved as epoch increases (1~10). However, 10 epoch accuracy = 0.6649, loss = 0.9317, it is not good enough.

### To do
- [ ] Checking if adding convolution layer or dense layer can further increase the accuracy.
- [ ] What does learning_rate do in Sequential and how does it affect the accuracy?
- [ ] What does the loss "sparse_categorical_crossentropy" do here? How do different loss function affect?
- [ ] What is the reduceLR?

**Comment by wooo0214** [comment]
Reading:
- arguments of Keras.layers.Conv2D
https://blog.csdn.net/Checkmate9949/article/details/119609758
   - try to add more filters in two convolution layers, like 16/32 instead of 6/16
   - try to add one conv layer or dense layer, arguments tbc

**Comment by wooo0214** [comment]
> Reading:
> 
> * arguments of Keras.layers.Conv2D
>   https://blog.csdn.net/Checkmate9949/article/details/119609758
>   
>   * 1. try to add more filters in two convolution layers, like 16/32 instead of 6/16
>   * 2. try to add one conv layer or dense layer, arguments tbc

a. okay, after changing the filter to 16 & 32 in 1st & 2nd conv2D layers, the accuracy and loss after 10 epochs were both slightly improved: accuracy is 0.7454, loss is 0.7341. Goal is: loss: 0.2577 - accuracy: 0.9099
> Accuracy isn't always the best metric to evaluate a model. How do you evaluate a model: https://blog.csdn.net/qq_19672707/article/details/88983814

**Comment by wooo0214** [comment]
Actually compared to the baseline where only two dense layers are added to fit the model, the accuracy was greatly improved after adding convolution layers (baseline: accuracy: 0.1488 - loss: 2.2392 in 3rd epochs. And its' too slow to execute so I won't fit more epochs)
